src/client/modules/content/widgets/authorization_widget/ProjectAuthorizationWidget.jsx

```diff
@@ -79,18 +103,18 @@ export default class ProjectAuthorizationWidget extends React.Component {
             this.state.valuesInMultiselect.split(',').forEach(userOrOrgName => {
                 promiseArrayToGrant.push(
                     this.props.restClient.projects.grantRightsToProject(this.props.ownerId,
-                        this.props.projectName,
-                        userOrOrgName,
-                        projectRights));
+                                                                        this.props.projectName,
+                                                                        userOrOrgName,
+                                                                        projectRights));
             });
         }
 
         Promise.all(promiseArrayToGrant)
             .then(() => {
                 this.props.refreshTable();
             })
-            .catch(() => {
-                console.log('Authorization denied.'); // eslint-disable-line no-console
+            .catch(err => {
+                console.log('Authorization denied! Error: ', err); // eslint-disable-line no-console
```

Comment:
Use `console.error` for errors. You don't need the `Errror: ` portion either, since err is of type `Error` the first line include the type, see below:
```
console.error(new Error('Something bad happened'));
Error: Something bad happened(…)(anonymous function)
```